package.json

```diff
@@ -39,12 +39,11 @@
     "@storybook/react": "^6.2.6",
     "@storybook/storybook-deployer": "^2.8.7",
     "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/react-hooks": "^8.0.0",
```

Comment:
`@testing-library/react-hooks` is now deprecated since `@testing-library/react` ship it's own version of `renderHook`